static/src/components/site-header/site-header.ts

```diff
@@ -232,6 +282,54 @@ export class SiteHeaderDropdown {
         });
     }
 
+    protected handleArrowDown() {
```

Comment:
Are you 100% sure that they code needs to be repeated?  I would assume it doesn't need to be in the side header class and only need to be here in the dropdown class.  Please review and confirm.